packages/vue-app/template/client.js

```diff
@@ -490,6 +490,14 @@ function fixPrepatch(to, ___) {
 }
 
 function nuxtReady(_app) {
+  <% if (store) { %>
+  if (app.store._actions && app.store._actions.nuxtClientInit) {
+    app.store.dispatch('nuxtClientInit', app.context)
```

Comment:
We should await the dispatch here, shouldn't we?

Reply:
@manniL you're too fast :) check again!